config/package-bundle/config/values-schema.yml

```diff
@@ -0,0 +1,20 @@
+#@data/values-schema
+---
+#@schema/desc "The namespace in which to deploy secretgen-controller"
+namespace: secretgen-controller
+#@schema/desc "Whether to create namespace specified for secretgen-controller"
+create_namespace: true
+
+#@schema/desc "Configuration for secretgen-controller deployment"
+#@overlay/match-child-defaults missing_ok=True
+deployment:
+  #@schema/desc "Update strategy of deployments, either 'RollingUpdate' or empty (use default strategy)"
+  updateStrategy: "RollingUpdate"
+  rollingUpdate:
+    #@schema/desc "The maxUnavailable of rollingUpdate. Applied only if RollingUpdate is used as updateStrategy"
+    maxUnavailable: 1
+    #@schema/desc "The maxSurge of rollingUpdate. Applied only if RollingUpdate is used as updateStrategy"
+    maxSurge: 0
```

Comment:
seems like we do not set them in https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/secretgen-controller/0.9.1/bundle/config/values.yaml#L6-L9. so this would be a change. i think we should not set updateStrategy.

Reply:
Good find, I can't recall why I set this one ...